src/dom-parts/ChildNodePart.js

```diff
@@ -59,8 +59,10 @@ const diffChildNodes = function (parentNode, domChildNodes, templateChildNodes,
         // If the DOM node doesn't exist, append/copy the template node
         if (domChildNode === undefined) {
             if (templateChildNode instanceof TemplatePart) {
+                // TODO: this should hopefully not happen anymore because we unwrap arrays in processNodePart ?!
```

Comment:
do wen need these todos or is this actually solved by the unwrap ?

Reply:
Yes! I have removed it now entirely